PostProcessing/IntegralsPlot.py

```diff
@@ -143,26 +67,25 @@ def main():
         linestyles = None
 
     if not len(linestyles) == len(plot_variables) and linestyles != None:
```

Comment:
It's a little hard to follow these conditionals

also, comparisons with `None` should be done with `is` instead of `==`

Reply:
fixed in newest commit https://github.com/ShaunFell/GRBoondi/pull/18/commits/edcbb92ca1e7867895adebd7bbcd814aab825162